server/src/event/event.service.ts

```diff
@@ -381,6 +392,12 @@ export class EventService {
   }
 
   async upsertEventFromDto(event: EventDto) {
+    const firstChal = await this.prisma.challenge.findFirst({
+      where: { id: event.challengeIds[0] },
+      select: { latitude: true, longitude: true },
+    });
+    console.log(firstChal);
```

Comment:
Do we want to have console.logs in the production code?

Reply:
No we do not